plugins/Terminal/js/utils/helpers.js

```diff
@@ -7,7 +7,7 @@ import url from 'url'
 import * as constants from '../constants/helper.js'
 
 export const checkSiaPath = () => new Promise((resolve, reject) => {
-	fs.stat(Path.join(SiaAPI.config.attr('siad').path, process.platform === 'win32' ? '../siac.exe' : '../siac'), (err) => {
+	fs.stat(Path.resolve(SiaAPI.config.attr('siac').path, './siac'), (err) => {
```

Comment:
Looks like `siac.path` is interpreted as the path of the _directory_ containing siac in the terminal plugin.  `siad.path` refers to the absolute path to the siad binary in other parts of the code, so `siac.path` should do the same